scoresheets/StoringGroceries.tex

```diff
@@ -19,12 +19,10 @@
 	\penaltyitem[5]{-60}{Storing an object without categorizing it correctly}
 
 	\scoreheading{Deus Ex Machina Penalties}
-	\penaltyitem[5]{-15}{A human handing an object over to the robot (the object is clearly indicated by the robot)}
-	\penaltyitem[5]{-30}{A human handing an object over to the robot}
-	\penaltyitem[5]{-45}{A human placing an object in the cabinet at a location clearly indicated by the robot}
-	\penaltyitem[5]{-90}{A human placing an object in the cabinet}
-	\penaltyitem[5]{-30}{A human placing an object in the cabinet at a location clearly indicated by the robot}
-	\penaltyitem[5]{-45}{A human pointing at a target location}
+	\penaltyitem[5]{-50}{A human handing an object over to the robot}
+	\penaltyitem[5]{-15}{A human placing an object in the cabinet}
+	\penaltyitem[5]{-30}{A human placing an object in the cabinet next to similar objects}
```

Comment:
"A human placing an object in the cabinet next to similar objects" is -30 (robot placing is +50) which is different from what @johaq suggested. Is it intended?

Reply:
Yes should be -50 imo.